### DISCUSSION TITLE
[8.0 QA] Subcomponent

### DISCUSSION THREAD

**Discussion by vanessayuenn** [storybookjs/storybook#25691] [Announcement]
#### Feature owner: @JReinhold 
Please comment in this discussion thread for any bug report and feedback for this feature as you test out Storybook 8. Kindly adhere to the following format:

> **Project**: Name of your project as entered in Notion
> **Version**: The version of storybook you tested with
> **Bug/feedback**: Details of the bugs/feedback you have

Feature owner should triage the comments here frequently and open issues as needed.

**Comment by valentinpalkovic** [comment]
**Project**: Bitwarden Clients
**Version**: 8.0.0-alpha.14
**Bug/feedback**: Subcomponents not working

Adding subcomponents doesn't lead to tabs on the docs page. They are not appearing.

**Reply by JReinhold** [reply]
@valentinpalkovic could you try adding the moduleMetadata as documented here: https://github.com/storybookjs/storybook/pull/25800/files#diff-48681eb0eb9cd8b27d2800299acf83efa1312fd6c35654c8355ece56fe430980

This is from the old subcomponents documentation, so this might be necessary for angular.
